lib/json_.js

```diff
@@ -21,22 +21,18 @@
         });
     }
 
-    JSON_.parse = function parse(text, reviver) {
-        text = text.replace(/"([^"]*)"\s*:/g, snakeToCamelCase);
-        return JSON.parse(text, reviver);
+    JSON_.parse = function parse(responseOrText, reviver) {
+        if (typeof responseOrText === 'string') {
+            var text = responseOrText.replace(/"([^"]*)"\s*:/g, snakeToCamelCase);
+            return JSON.parse(text, reviver);
+        }
+        return responseOrText.text().then(parse); // Assume text is a Response object from fetch.
     };
 
     JSON_.stringify = function stringify(value, replacer, space) {
         var str = JSON.stringify(value, replacer, space);
         return str.replace(/"([^"]*)"\s*:/g, camelToSnakeCase);
     };
 
-    // Export to popular environments boilerplate.
-    if (typeof define === "function" && define.amd) {
-        define(JSON_);
-    } else if (typeof module !== "undefined" && module.exports) {
-        module.exports = JSON_;
-    } else {
-        window["JSON_"] = JSON_;
-    }
-}());
+    module.exports = JSON_;
```

Comment:
I can see no longer supporting AMD... fuck dat. But why also kill non module envs (jsfiddle for example)